game/scripts/npc/abilities/treant_living_armor.txt

```diff
@@ -35,7 +35,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "total_heal"                                      "180 270 360 450 900 1800" //OAA
+        "total_heal"                                      "150 350 550 750 1500 3000" //OAA
```

Comment:
You are aware that 750 is too much. But it's not that big. Warlock for example has 700 heal at lvl 4 with the talent.